install.sh

```diff
@@ -211,9 +233,16 @@ verify_tarball() {
     fi
 }
 unpack_tarball() {
-    info "unpacking tarball file"
-    mkdir -p /usr/local
-    tar xzf $TMP_TARBALL -C /usr/local
+    info "unpacking tarball file to ${INSTALL_RKE2_TAR_DIR}"
+    mkdir -p ${INSTALL_RKE2_TAR_DIR}
+    tar xzf $TMP_TARBALL -C ${INSTALL_RKE2_TAR_DIR}
+    if [ "${INSTALL_RKE2_TAR_DIR}" != "/usr/local" ]; then
```

Comment:
Small nit but can we define a var for this and reference that throughout this script?

Reply:
done!